src/tests/admin/conferenceroomoutoforderplanning/OutOfOrderIsCreatedIfDescriptionIsEmpty.java

```diff
@@ -0,0 +1,58 @@
+package tests.admin.conferenceroomoutoforderplanning;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.json.JSONException;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomOutOfOrderPlanningPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC17: Verify an Out Of Order can be created if description text box is empty
+ * @author Yesica Acha
+ *
+ */
+public class OutOfOrderIsCreatedIfDescriptionIsEmpty {
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	List<Map<String, String>> testData = excelReader.getMapValues("OutOfOrderPlanning");
+	String roomName = testData.get(8).get("Room Name");
+	String title = testData.get(8).get("Title");
```

Comment:
As well here, keep only roomName and title as global variables, others declare inside the test method